frame/contracts/src/weights.rs

```diff
@@ -204,11 +204,11 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts PristineCode (r:0 w:1)
 	// Storage: Contracts OwnerInfoOf (r:0 w:1)
 	fn instantiate_with_code(c: u32, s: u32, ) -> Weight {
-		(210_420_000 as Weight)
-			// Standard Error: 133_000
-			.saturating_add((145_601_000 as Weight).saturating_mul(c as Weight))
-			// Standard Error: 8_000
-			.saturating_add((1_760_000 as Weight).saturating_mul(s as Weight))
+		(312_769_000 as Weight)
+			// Standard Error: 122_000
+			.saturating_add((142_000_000 as Weight).saturating_mul(c as Weight))
+			// Standard Error: 7_000
+			.saturating_add((1_798_000 as Weight).saturating_mul(s as Weight))
```

Comment:
The overhead of instantiating per code byte seems to stay the same. General overhead goes up. I assume setting up the wasm instance is slower now.